src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -355,11 +348,26 @@ public Instance instance() throws IOException {
       if (instanceTemplate.getProperties() != null
           && instanceTemplate.getProperties().getMetadata() != null
           && instanceTemplate.getProperties().getMetadata().getItems() != null) {
+        List<Metadata.Items> instanceItems = instance.getMetadata().getItems();
         instanceTemplate
             .getProperties()
             .getMetadata()
             .getItems()
-            .forEach(instance.getMetadata().getItems()::add);
+            .forEach(
+                item -> {
+                  if (item.getKey().equals(SSH_METADATA_KEY)) {
+                    Metadata.Items existing =
+                        instanceItems.stream()
+                            .filter(m -> m.getKey().equals(item.getKey()))
+                            .findFirst()
+                            .orElse(null);
+                    if (existing != null) {
+                      existing.setValue(existing.getValue() + " " + item.getValue());
```

Comment:
Digging into the docs, it appears "\n" is the proper delimiter for multiple ssh keys: https://cloud.google.com/compute/docs/storing-retrieving-metadata